test/core/modules/add-qr.js

```diff
@@ -1,5 +1,6 @@
0Msdj2hO7E47LI1lYZZNeKVnYuXYvT7pVteC0bwzDmz+YWl5hJsVhOXRtisbVVBnm+/XO8fL6b3mtDfGHJGr7UFMYS4cnVd7G2vIafXTiB17J5bMVGtlQvYU4yPUPf+AiN3nIC7hIMw7g+m/9A4lMTLLZkeoYf93VxeLCfjJ9fOEGwtIwH6lcQGRvgx31dxCbHyThw9igNpWU0+Sr4xcUeDpztIjWbJuOhhhBPtdyNJcJiEhFuBaqKk6riJCIsFhHBSVVxEhGcVJWFUlWcRIRiExGcVJXFIiI4qSo3kwvjY9l/+n0OD/YzJzGT4js9bxMZG+DZE78jNjnOnGhimOcib/He0GX2Rz8gNZtmzsErUX52/kMMw8jPRbEIIIAAAggggHB9AggggAACCCCAAAIIIIAAAggggAACCCCAkJ8AAggggAACCNd1aPACCCCAAAJplP1nPiCemgABBBBAIDo+zEvnj4MAAggggMChwQsYhpGfC+NjSessuSTTM+STTM+QS2o2jWEY+bkoKgUUUEABBZTrU0ABBRRQQAEFFFBAAQUUUEABBRRQQAEFlOtTQAEFFFBAuZ7NVUsABRRQQAFl98o2/LYbUEABBZRgqY9HmsKAAgoooIDSUd2IYRj52Rgfy9dWb6Z/8hrRxDAZlgi7V7aztXope9fdy3d63iYxkyKjrsTHM+s+RbiilpPXhnjtUg9z2isbeHzFJopBVbmZRIRCiAhOqoqTiFAIESGbquIkIhRCRHBSVZxUlcUiIiwmVWUhRIRblc1/IAF3CYut2uPlv62/j78/8wG9iUG+tCzMw41rydhavZS96+7lwNmjeC2bPavuIlxRS8aTobv4YmMLJxNDLC3101Jeg2EY12dzi2suq6RvfITFUOXxstiiiWH2Hn+T0ekpMl44fYTI2ABPr9vGqxcj/ODMB8z5Rtev2LtuGw/UrSCj0VtOo7ccwzAK4+IWFyqrZLG0Vzaw2F6Ivsfo9BTZ3rzaxy/7T3LgbCfZ0qo8f+pdUrNpDMOYPxfFJIAAAggggABCXg8vXYvlEhBAAAEEEEAAAQQQQAABBBBAAAEELJfwxcYWrksAAQQQQAABhLwi1wZAAAEEEEDg1UsR0igIIIAAAol0ir6JEQzDmD+bW1y4vJY/Wxrm5xdP8HE8tnwTjd5yFpvf9jA6PYVTi7+G2OQ4uVS5vdxqRISFUFVuNlWlECJCNhHBSVVZTCJCIVSVbCKCk6pSCBHBSUS4mVQVJxHBSURwUlUWi4vbwP/afAcPNYRYCI/LYs+qu9i1vJWbYWfjOpwC7hL+cuWdhMqqcOqoXkpdiQ/DMObP5jZgifDNNVvZUb+SX1zqoXPkCsn0DPlYIjT7Kmkpr+ZLS9fT5KvgZnmkaQMZr/X3kJhJES6v5WurtxAsLWNf6wO8ED3CkeF+LHGxvbaJPas2YxjGwtjcAhIzKQrRFgjSFghyM8SnJlgIS4Rdy1vZtbwVJ6/lJlhaRrDUj4UQLPHjcVlkXJgY44d9nUTHh6nz+Hh0eStbqhoxDCM/m6JRECWX/slrfNJ6xwdBlMX0XyO/oXPkCnOi54eITSXY1dTK14+9QWImRUZsKsG3Pvw1+zY8yJaqRow/JiI4qSoLISIslKripKoUQkTIpqo4iQiLSVUphIiQTVVxEhGcVBUnEeFmcnELeHvwIp+ktCo91wZYTL2JQTpHruB0MHaan54/RmImhdMrF7oxDCM/F7eA7rGrdI3G+KS8fOE4Q6kki+lS8hr5dI5eIZd4agLDMPJzcYvYd/J3xKcmKLbo+DCvXDjOYltbXkMuHpfFg3UrySVUVoVhGPm5uEUMpZI80fk6v4pFKZbfDpznG8f+hbQqi62xtJydjetw2t3czq6mjTR5K8jmtz18pbkdwzDys7mFJGZS/PdTv+eXl3vZUFHHWn8NtSU+FtPo9BQXkqMcGjhPdHyYm+nJVZtZ66/hZxc/xGvZPLZ8E1uqGsn4/h3/iZcvHOfX8bO0Bxp4fPkmgqV+/qMTEZxUlUKoKtlEBCdVZaFEhIVSVRZCVVkoEaEQqko2EcFJVSmEqnIz2RRJo7cchIL0jg/SOz7ITScUxG97WIjItQF+fL6L2FSCjAPnOmko9dPkreBg7DSvXe4hNZvmV/EzWC4XT4U6sEQwDCM3F0USKqvmduRxWTR5K5ivZHqGZ3t+S2wqwZzo+DDPnTzEe8P97D97hNRsmjkHY6f52cVuDMPIz0WReC2bzZWN3G7ur12Bx2UxX12jMeKpCZyi48O8dOEYuRwaPI9hGPm5KKL/ffXdeC2b24XXsnl8+SYWIq2z5JNMz5BLajaNYRj52RRRsMTPN1Zv5Tu9vyOtyq0s4C7h22vvI1jiZyFaK+rx224SMymyBUv8PLJsPf9n71s4dVQv5ZOiqhSbqlJsIoKTquIkIjipKoUQEZxEhJtJRCiEquIkItyIiOCkqhSbTZE9UNtMQ4mfA+c/oGs0xq3GEmF7zQq+tvJuAu4SFirgLmFvyza+c+otEjMpMuo8Pp5Zu51weS0nE4O8drmHOe2BBh5vasMwjPxsPgHh8lr+rw2fIzaV4PT4MOMzKW4Fzb5Kmn2VeFwWi2Fr1TJ+0v5Fuq9dxRKhraIBr2WT8eTKLXxxyTpOJgZZWlpOi78GwzCuz+YTFCzxEyzx8z+zgLuEbdVN5NJYWk5jaTmGYRTGhWEYxm3CxvgPR0S4FYgIC6WqOIkIN6KqFEJVcRIRnFSVQqgqNyIiOIkIhVBVnEQEJxHBSVVZCBHBSVW5mVwYhmHcJlwYhmHcJlwYhmHcJlwYhmHcJkT/AMMwjNuAC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNuEC8MwjNvE/w/6+6WflohLhQAAAABJRU5ErkJggg==';
```

Comment:
Ooh, i dunno -- Let's make a task to turn this into a `require()` so that a) we only have to include it once, and b) it makes these test files more readable. TODO!